patroni/ha.py

```diff
@@ -242,6 +245,11 @@ def process_sync_replication(self):
                     if not self.dcs.write_sync_state(self.state_handler.name, None, index=self.cluster.sync.index):
                         logger.info('Synchronous replication key updated by someone else.')
                         return
+
+                logger.info("Debug: %s", self.is_synchronous_mode_strict())
+                if self.is_synchronous_mode_strict() and picked is None:
+                    picked = 'patroni_dummy_host'
```

Comment:
* would be different - replication to any active host (even though there are none at the moment).

I used 'patroni_dummy_host' to guarantee replication to no hosts. ' ' could be used - but that would be easy to miss when debugging.

At this time we have no hosts - but we also want to assert that if one does return we don't want to use it until such time as Patroni can do a forced setting (otherwise we negate the benefit of synchronous_mode and don't know where our sync writes are going).